### PR TITLE
Add folder to library settings

### LINKED ISSUE #50: Display library root folder path in settings

### PR DESCRIPTION
**Pull request by ChappIO**
Closes #50

### CODE REVIEW

File: web/src/pages/settings/ManageLibrariesModal.tsx [hunk 1]
```diff
@@ -97,6 +97,7 @@ const EditLibraryForm: React.FC<{ library: LibraryDetails; onDone: () => void }>
         submitLabel="Save"
       >
         <TextInput required label="Name:" value={name} onChange={setName} id="name" />
+        <TextInput tabIndex={-1} required label="Folder:" value={library.rootFolder} id="folder" />
```
Comment (LucaScorpion) [inline web/src/pages/settings/ManageLibrariesModal.tsx:100]: Should we default `tabIndex` to -1 for readonly fields? Or would there be a reason to have a readonly field you _can_ tab-navigate to?

Comment (ChappIO) [reply web/src/pages/settings/ManageLibrariesModal.tsx]: Nothing gets past you. Done.

Comment (LucaScorpion) [reply web/src/pages/settings/ManageLibrariesModal.tsx]: Uhmm, I think you did it the wrong way around ;P

```
tabIndex={onChange && -1}
```

means you can _only_ tab to readonly fields :sweat_smile:

Comment (ChappIO) [reply web/src/pages/settings/ManageLibrariesModal.tsx]: heheh, oops

Comment (ChappIO) [reply web/src/pages/settings/ManageLibrariesModal.tsx]: Fixed

**Review by LucaScorpion — APPROVED** [review]

**Pull request merged**